bower.json

```diff
@@ -20,11 +20,11 @@
     "jquery": "~2.1.4",
     "angular-highlightjs": "~0.4.3",
```

Comment:
Try updating this to 0.6.1, it looks like VersionEye is claiming that 0.4.3 has no license.